hazelcast/src/test/java/com/hazelcast/map/impl/recordstore/LazyEvictableEntryViewTest.java

```diff
@@ -42,7 +42,8 @@
 @Category({QuickTest.class, ParallelJVMTest.class})
 public class LazyEvictableEntryViewTest {
 
-    private static final int ENTRY_VIEW_COST_IN_BYTES = 77 + 3 * REFERENCE_COST_IN_BYTES;
+    private static final int WITH_COMPRESSED_OOPS_ENTRY_VIEW_COST_IN_BYTES = 89;
```

Comment:
Is the change related to some recent update to the record layout?
And how are these calculated (89 = 77+3*4 but 105=?)?

Reply:
It was caused by a latest [change](https://github.com/hazelcast/hazelcast/commit/fbca5a3bedcf8d0bb4b13bd5ae690ac794ee538d#diff-c22c3cf71c9f86aad67ede31d6ade8f9R45).
But there was no record format change, it was a change in testing.